docs/Formula-Cookbook.md

```diff
@@ -514,6 +514,16 @@ The `std_*_args` methods, as well as the arguments they pass, are:
 "--wrap-mode=nofallback"
 ```
 
+#### `std_npm_args`
+
+```ruby
+"require 'language/node'"
+
+"return Language::Node.std_npm_install_args(Pathname(prefix)) if prefix"
+
+"Language::Node.local_npm_install_args"
```

Comment:
I would list what these actually resolve to:

```ruby
"-ddd"
"--global"
"--build-from-source"
"--cache=$(brew --cache)/npm_cache"
"--prefix=#{libexec}"
"$PWD/$(npm pack 2>/dev/null)"
```

not sure the best way to express that last line though